src/hooks/useSubmitForm/useSubmitForm.ts

```diff
@@ -51,5 +52,6 @@ export const useSubmitForm = (activity: Activity): UseFormActivityHook => {
   return {
     onSubmit,
     disabled: isSubmitting,
+    isSubmitting,
```

Comment:
we already return this in a line above as disabled prop.

Reply:
yes, but I assumed it to be internal logic of the process to decide when should the submitting be disabled.

I'll change it 👍